### DISCUSSION TITLE
iOS Shortcut for adding to yt-dlp Web UI queue

### DISCUSSION THREAD

**Discussion by joestump** [marcopeocchi/yt-dlp-web-ui#229] [Show and tell]
I've created an iOS shortcut that will send URLs from the share sheet to the `/api/v1/exec` endpoint of your `yt-dlp` Web UI instance: https://www.icloud.com/shortcuts/b7d88cd00533413db6d90ef8e7544344

It will ask for your instance's base URL (e.g. `https://yt-dlp.example.internal`) on install. After that, you just click "Send to yt-dlp Web UI" from the share sheet. If it's a valid URL, `yt-dlp` will do it's thing.

**Comment by L041S** [comment]
Hello, and first of all, thank you for the shortcut, it works perfectly!
It seems that the shortcut downloads the best quality by default (in my tests, it uses the WebM format).
Would it be possible to add an option in the shortcut to use a template from within the yt-dl web UI for example use mp4 and limit to 1080p max?
